experimental/FTheoryTools/src/TateModels/constructors.jl

```diff
@@ -61,7 +61,7 @@ function global_tate_model(base::NormalToricVariety,
   vs2 = collect(keys(defining_section_parametrization))
   @req all(in(["a1", "a2", "a3", "a4", "a6"]), vs2) "Only the Tate sections a1, a2, a3, a4, a6 must be parametrized"
   
-  gens_base_names = [string(g) for g in gens(cox_ring(base))]
+  gens_base_names = [string(g) for g in symbols(cox_ring(base))]
```

Comment:
@HereAround pinging you to make you aware that `symbols(::MPolyRing)` exists. In many of these cases you probably don't even need to convert to strings afterwards, but I kept it like this if I didn't see an obvious way to change the below code